Pipfile

```diff
@@ -6,10 +6,14 @@ verify_ssl = true
 [dev-packages]
 bandit = "==1.4.0"
 coveralls = "==1.2.0"
+cython = "==0.29.2"
```

Comment:
Does `cython` have to be included explicitly, instead of getting picked up as a dependency? I don't see `import cython` anywhere in our code, so I'd prefer not to include and pin it explicitly if possible.

Reply:
This has been an issue with pymssql not picking up the dependency. I'll double check here that it's still an issue though.

Reply:
Yeah I asked around and it seems like a few other people have seen it as well. Let's leave it in.